digdag-tests/src/test/java/acceptance/td/TdIT.java

```diff
@@ -114,6 +135,108 @@ public void testRunQuery()
     }
 
     @Test
+    public void testRunQueryWithEnvProxy()
+            throws Exception
+    {
+        List<FullHttpRequest> jobIssueRequests = Collections.synchronizedList(new ArrayList<>());
+
+        proxyServer = DefaultHttpProxyServer
```

Comment:
How about moving these codes to a new private method that returns HttpProxyServerBootstrap and calling it from this method and testRunQueryOnServerWithEnvProxy() to reduce duplicated codes?